scripts/run-dependencies.sh

```diff
@@ -17,7 +17,7 @@ docker run --rm -d \
   --network=$NETWORK \
   --name=$API \
   -p 8888:8888 \
-  mrcide/hintr:mrc-418
+  mrcide/hintr:mrc-418-rgf
```

Comment:
hintr:latest still broken? Build is green..

Reply:
The bug was with the tests! But in fact this branch has now been merged :+1: